### PR TITLE
specification: Define CoVE function ID namespaces

### LINKED ISSUE #39: CoVE-IO FID namespace
The [CoVE-IO](https://github.com/riscv-non-isa/riscv-ap-tee-io) specification extends the CoVE ABI for supporting device direct assignment into TVMs.

The CoVE-IO ABI extensions require their own SBI FIDs and are currently left TBD in the [spec](https://github.com/riscv-non-isa/riscv-ap-tee-io/blob/main/specification/09-coveio_abi.adoc). I propose that we assign a specific FID range for CoVE-IO, so that the CoVE and CoVE-IO FIDs can be formally defined without overlapping or interleaving.

Since FIDs can fill an entire register (`a6`), one possibility would be to reserve the top half `COVG` and `COVH` EID spaces for CoVE extensions. CoVE-IO could e.g take 6 bits from there and use them as needed.

@rsahita @jyao1 WDYT?

### PR DESCRIPTION
**Pull request by sameo**
Fixes #39

### CODE REVIEW

**Comment by sameo** [comment]
cc @rsahita @jyao1

File: specification/sbi_cove.adoc [hunk 1]
```diff
@@ -9,6 +9,28 @@ extension. This specification introduces three new extensions:
...
+
+[#table_cove_fid_namespaces]
+.CoVE FID Namespaces
+[cols="2,2,6", width=90%, align="center"]
+|===
+2+^s|  SBI Function ID Range                .2+^s| Specification Namespace
+  ^| Start          ^| End
+   | 0 (`0x0`)       | 1023 (`0x3ff`)         | CoVE
+   | 1024 (`0x400`)  | 1087 (`0x43f`)         | CoVE-IO
+|===
```
Comment (rsahita) [inline specification/sbi_cove.adoc:32]: FIDs over 0x43f may be used for other future extensions right? should add that row and specify that range as unallocated.

**Review by rsahita — APPROVED** [review]
LGTM with comments addressed - thanks

File: specification/sbi_cove.adoc (hunk 1 above)
Comment (sameo) [reply specification/sbi_cove.adoc:33]: Right, good point. This is fixed now.

**Review by rsahita — APPROVED** [review]

**Pull request merged**